Fixed require statements

Trial97 opened the pull request.

lcpz commented on widget/bat.lua:
```diff
@@ -22,7 +22,7 @@ local function factory(args)
 
     if not fs.is_dir(pspath) then
-        naughty.notify { text = "lain.widget.bat: invalid power supply path", timeout = 0 }
+        naughty.notify { text = tostring(...):match(".*lain") .. ".widget.bat: invalid power supply path", timeout = 0 }
```
`naughty.notify` is not vararg, so this raises the following error:

```lua
cannot use '...' outside a vararg function near '...'
```

Since we cannot touch `naughty.notify`, please revert this change.

lcpz commented:
Thank you for your work. Please see my review.

Before merging, can please you ensure that you tested every widget, layout and utility?

Maybe you can help me with #527 as well.

Trial97 commented:
Thanks for the review. I reverted the change.
Regarding testing everything it will take me some time as I just started learning lua and awesomewm.
As regards #527 can you elaborate?
It is enough for the files to only have the `require` statement in them?
e.g.lain-test-quake.lua
```
local lain      = require('extra.lain')
return lain.util.quake({
    app = 'kitty',
    name = "QuakeDD",
    argname = "--name %s",
    followtag = true,
    settings = function(c)  ...    end
    extra = '',
    border = 1,
    visible = false,
    overlap = false,
    screen = awful.screen.focused(),
    height = 0.25,
    width = 1,
    vert = "top",
    horiz = "left"
})
```

Trial97 commented:
Ok, a small update to this:
- added a first set of tests(not all files but maybe)
- fixed requires to work with both  tests and also in an awesome config

Let me know if there are some changes that need to be done regarding tests(if not I will continue adding them in the same manner if I have time)
